docs/api_documentation.md

```diff
@@ -200,15 +200,17 @@ details.
 
 ### Supported scopes
 
-OSIAM implements the different http methods as single scopes plus the scope 'email':
+OSIAM implements the different http methods as single scopes plus the scope
+'email':
```

Comment:
I will change the text. what do you think about

`OSIAM using the following HTTP methods and maps them to scopes.`